c/vsam.c

```diff
@@ -31,6 +31,7 @@
 #include "utils.h" 
 #include "qsam.h"  
 #include "vsam.h"
+#include "logging.h"
```

Comment:
All the messages in this file seem like trace messages. I think we should just keep TRACE. No need to use zowelog.